psl/schema-ast/src/ast/argument.rs

```diff
@@ -68,6 +68,13 @@ impl Argument {
     pub fn is_unnamed(&self) -> bool {
         self.name.is_none()
     }
+
+    pub fn name(&self) -> Option<&str> {
+        match &self.name {
+            Some(ident) => Some(ident.name.as_str()),
+            None => None,
+        }
+    }
```

Comment:
Perhaps could you implement the `WithName` trait instead

Reply:
The problem there is that we're not guaranteed to have a name specified for arguments. We _could_ decide to just return an empty string so that we could use an impl, but having it specifically be an `Option` made more sense to me.

Reply:
Ah, right, I missed the Option<&str> part. All good, this is fine the way it is 👍